### PR TITLE
Editorial: Link to whatwg/meta COMMITING.md

### PR DESCRIPTION
**Pull request by snuggs**
Addresses:
    - https://github.com/whatwg/meta/issues/65

### CODE REVIEW

**Comment by annevk** [comment]
Should this also reference https://github.com/whatwg/meta/blob/master/CONTRIBUTING.md? And be renamed in uppercase for consistency?

**Comment by snuggs** [comment]
@annevk this should be more to our liking, yes?

**Comment by snuggs** [comment]
Also an aside nit. Would changing a reference link be considered `Meta` or `Editorial`? Probably splitting hairs but would rather follow the "in" crowd. 😎

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
-# Contribution guidelines for the Streams Standard
+# Streams Standard contributor guidelines
+
+These are the guidelines for contributing to the Streams Standard. First see the [WHATWG contributor guidelines](https://github.com/whatwg/meta/blob/master/CONTRIBUTING.md).
+
+We label [good first issues](https://github.com/whatwg/html/labels/good%20first%20issue) that you could help us fix, to get a taste for how to submit pull requests, how the build process works, and so on.
```
Comment (domenic) [inline CONTRIBUTING.md]: This should be streams, not html

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+
+## Pull Requests
+
+In short, change `source` and submit your patch. Try to follow the [spec editorial conventions](#spec-editorial-conventions) below.
```
Comment (domenic) [inline CONTRIBUTING.md]: The `source` file does not exist here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,4 +1,18 @@
-# Contribution guidelines for the Streams Standard
+# Streams Standard contributor guidelines
+
+These are the guidelines for contributing to the Streams Standard. First see the [WHATWG contributor guidelines](https://github.com/whatwg/meta/blob/master/CONTRIBUTING.md).
+
+We label [good first issues](https://github.com/whatwg/html/labels/good%20first%20issue) that you could help us fix, to get a taste for how to submit pull requests, how the build process works, and so on.
+
+## Pull Requests
```
Comment (domenic) [inline CONTRIBUTING.md]: Lowercase "r" for consistency

**Comment by snuggs** [comment]
@domenic should look better to our liking. Thanks for the nits.

P.S. I signed the participation agreement. Hasn't propagated yet tho. /cc @annevk

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,4 +1,18 @@
-# Contribution guidelines for the Streams Standard
+# Streams Standard contributor guidelines
```
Comment (snuggs) [inline CONTRIBUTING.md]: @domenic "Streams standard"?

Comment (domenic) [reply CONTRIBUTING.md]: Nah, in this case "Streams Standard" is a proper noun.

**Pull request merged**